build-system/tasks/serve.js

```diff
@@ -66,6 +66,7 @@ let lazyBuild = false;
  */
 function getMiddleware() {
   const middleware = [require('../server/app')]; // Lazy-required to enable live-reload
+  middleware.push(header({'Access-Control-Allow-Origin': '*'}));
```

Comment:
I don't think `serve.js` should have to know about the details of implementation of how the server responds to a given request. Can we move this to `build-system/server/app.js` and insert a generic URL handler that adds the header to all requests? Between lines 74 and 76 might work:

https://github.com/ampproject/amphtml/blob/6616e578b8c257d82ab91fd99aa720fcdc4022b1/build-system/server/app.js#L59-L86

Reply:
Aah, I now see why you initially added the header here. These lines (written by me a long time ago) set a bad example :)

https://github.com/ampproject/amphtml/blob/9cb59cef81be068ef7179568f335d7fc0e66c390/build-system/tasks/serve.js#L72-L74

Probably makes sense to move this to `app.js` as well, but feel free to punt / ignore if you don't think it makes sense.